src/clj/rems/css/styles.clj

```diff
@@ -555,6 +566,37 @@
                    :flex-direction "row"
                    :justify-content "space-between"
                    :min-width "100%"}]
+
+   ;; Logo, login, etc.
+   [:.logo-menu {:height logo-height-menu
+                 :background-color (get-theme-attribute :logo-bgcolor)
+                 :width "100px"}]
+   [:.logo {:height logo-height
+            :background-color (get-theme-attribute :logo-bgcolor)
+            :width "100%"
+            :margin "0 auto"
+            :padding "0 20px"
+            :margin-bottom (u/em 1)}]
+   [(s/descendant :.logo :.img) {:height "100%"
+                                 :background-color (get-theme-attribute :logo-bgcolor)
+                                 :background-image (get-logo-image context/*lang*)
+                                 :-webkit-background-size :contain
+                                 :-moz-o-background-size :contain
+                                 :-o-background-size :contain
+                                 :background-size :contain
+                                 :background-repeat :no-repeat
+                                 :background-position [[:center :center]]
+                                 :background-origin (get-theme-attribute :logo-content-origin)}]
+   [(s/descendant :.logo-menu :.img) {:height "100%"
+                                      :background-color (get-theme-attribute :logo-bgcolor)
+                                      :background-image (get-logo-navigation context/*lang*)
+                                      :-webkit-background-size :contain
+                                      :-moz-o-background-size :contain
+                                      :-o-background-size :contain
+                                      :background-size :contain
+                                      :background-repeat :no-repeat
+                                      :background-position [[:center :center]]
+                                      :background-origin (get-theme-attribute :logo-content-origin)}]
```

Comment:
We could remove some duplication from these two nearly identical blocks of styling.

Reply:
I was planning to submit the refactoring PR later, would that work for you?

Reply:
For this specific refactoring yes.